docs/operate/get-started/other-hardware/manage-modules.md

```diff
@@ -244,6 +244,24 @@ For more details, see the [`upload-module` GitHub Action documentation](https://
 
    Once the build is complete, the module will automatically update in the [Viam Registry](https://app.viam.com/registry), and the machines set to use the latest [version](/operate/reference/module-configuration/#module-versioning) of the module will automatically update to the new version.
 
+## Supported platforms for cloud-build
+
+When using cloud-build, you can specify which platforms you want to build your module for in the `arch` field of your `meta.json` file. The following table lists all available platforms:
+
+| Platform | Recommended | Supported in Cloud Build | Container Used by Cloud Build | Notes |
+|----------|-------------|--------------------------|------------------------------|-------|
+| `linux/amd64` | ✅ | ✅ | Ubuntu | Standard x86_64 Linux platform |
+| `linux/arm64` | ✅ | ✅ | Ubuntu | For ARM64 Linux devices like Raspberry Pi 4 |
+| `darwin/arm64` | ✅ | ✅ | macOS | For Apple Silicon Macs (M1/M2/M3) |
+| `linux/arm32v6` | ✅ | ❌ | N/A | For older ARM devices; must be built manually |
```

Comment:
What does the "Recommended" column imply? My impression is they stopped making armv6 devices in 2005ish; does anyone still have these any more? Why are they recommended?

Reply:
I think it implies whether we are committed to supporting it